Libraries/libcar/Headers/car/Rendition.h

```diff
@@ -72,20 +72,49 @@ class Rendition {
         { return _format; }
     };
 
+public:
+    enum resize_mode {
```

Comment:
Are these resizing modes a property of the car format? If so, might be better for `car_format.h`.

nit: you could use `enum class` here to use namespacing instead of prefixing for the values.